libs/blocks/mobile-app-banner/mobile-app-banner.js

```diff
@@ -13,8 +13,19 @@ async function getKey(product) {
   return keyMatch[0]?.key;
 }
 
+async function getECID() {
```

Comment:
We should get the ECID from `alloy`, not the cookie. Here's how we did it for Jarvis:
https://github.com/adobecom/milo/blob/stage/libs/features/jarvis-chat.js#L234

Reply:
Sure @narcis-radu , I made the suggested change. Thanks!